Armchair.xcodeproj/project.pbxproj

```diff
@@ -720,7 +718,7 @@
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 4.2;
```

Comment:
Do we need to update the swift version? Is this possibly problematic?

Reply:
Their project actually doesn't build without bumping. And their podspec specifies 4.2

Reply:
Weird... But sounds good!